.nvmrc

```diff
@@ -1 +1 @@
-v12.0.0
+v16.0.0
```

Comment:
This is a good call, we added the `engine` entry before adding our `.nvmrc`. Since both Netlify and the local environment use the `.nvmrc`, I think we can just go ahead and remove the `engine` entry from package.json, so that we don't have to worry about them getting out of sync.